### PR TITLE
Standardize C++ version naming

### PR DESCRIPTION
**Pull request by Rageking8**
The canonical way to represent a C++ version (only the last 2 digits of the year) is without the extra space character (i.e. either C++20 or C++ 2020). This PR removes all extra spaces as the canonical version is more prevalent in this docs repository and is used as such in the official ISO website.

### CODE REVIEW

File: docs/standard-library/chrono-operators.md [hunk 1]
```diff
@@ -603,56 +603,56 @@ constexpr bool operator==(
...
-constexpr bool operator==(const year_month_weekday& Left, const year_month_weekday& Right) noexcept; // C++ 20
+constexpr bool operator==(const year_month_weekday& Left, const year_month_weekday& Right) noexcept; // C++20
 
 // 17)  year_month_weekday_last
-constexpr bool operator==(const year_month_weekday_last& Left, const year_month_weekday_last& Right) noexcept; // C++ 20
+constexpr bool operator==(const year_month_weekday_last& Left, const year_month_weekday_last& Right) noexcept; // C++20
 
 // 18) time_zone_link
-bool operator==(const time_zone_link& Left, const time_zone_link& Right) noexcept;
+bool operator==(const time_zone_link& Left, const time_zone_link& Right) noexcept; // C++20
```
Comment (Rageking8) [inline docs/standard-library/chrono-operators.md:651]: This is missing the `C++20` label, added it as a drive-by.

**Comment by Jak-MS** [comment]
@TylerMSFT - is this yours to review? If not, you can reassign or let me know who to assign. Thanks!

- IMPORTANT: When this content is ready to merge, you must add `#sign-off` in a comment or the approval may get overlooked.


#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Review by TylerMSFT — APPROVED** [review]
Nice cleanup - thanks @rageking8

**Comment by TylerMSFT** [comment]
#sign-off
